<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tvOverlay</class>
 <widget class="QWidget" name="tvOverlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#tvOverlay{
background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="main_overlay" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>149</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>149</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#main_overlay{
background-color: rgb(71, 71, 71);
}</string>
     </property>
     <widget class="QLabel" name="chanel_pick">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>12</y>
        <width>119</width>
        <height>119</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Документы/it999_dark_logo/it999_dark_165x165/Первый канал.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="chanel">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>8</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 17pt &quot;Noto Looped Thai UI ExtraBold&quot;;
color: orange;</string>
      </property>
      <property name="text">
       <string>Первый канал</string>
      </property>
     </widget>
     <widget class="QLabel" name="current_programm">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>35</y>
        <width>581</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 100 20pt &quot;Noto Looped Thai UI ExtraBold&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>---EPG loading---</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>74</y>
        <width>429</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QProgressBar {
    	/* Outer border */
   
    }
     
     /* progress bar */ 
    QProgressBar::chunk {
    	 background-color: orange;
    }</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="start_time">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>71</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 100 17pt &quot;Noto Looped Thai UI ExtraBold&quot;;
color: orange;</string>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QLabel" name="end_time">
      <property name="geometry">
       <rect>
        <x>641</x>
        <y>72</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 100 17pt &quot;Noto Looped Thai UI ExtraBold&quot;;
color: orange;</string>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>95</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 100 17pt &quot;Noto Looped Thai UI ExtraBold&quot;;
color: orange;</string>
      </property>
      <property name="text">
       <string>Далее:</string>
      </property>
     </widget>
     <widget class="QLabel" name="next_programm">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>95</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 100 17pt &quot;Noto Looped Thai UI ExtraBold&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
